### PR TITLE
Add Windows installation; move installation in README to tutorial

### PR DESCRIPTION
**Pull request by mabelzhang**
Partially addresses https://github.com/ignitionrobotics/docs/issues/117 and https://github.com/ignitionrobotics/docs/issues/14#issuecomment-750431388 .

- Adds Windows binary and from source install instructions for https://github.com/ignitionrobotics/docs/issues/117 .
   For some reason, I didn't encounter the compilation error @Jshep1 mentioned here https://github.com/ignitionrobotics/docs/issues/96#issuecomment-742096017 .
   My steps are slightly different, as necessitated by these per-package installation tutorials.
   John built everything from source using `colcon`.
   I did `conda-forge` binary install for `ign-cmake` and `ign-math`, and then pure `cmake`.
   Not sure if that makes the difference or I'm missing something.

- Moves installation in README to a new installation tutorial page for https://github.com/ignitionrobotics/docs/issues/14#issuecomment-750431388 .
   TODO: The URL in README currently links to `ign-common` tutorials main page, since the installation tutorial webpage doesn't exist yet. Somebody should change the link after this PR is deployed. Trivial fix of the URL in README line 63.

### CODE REVIEW

**Comment by traversaro** [comment]
>  Not sure if that makes the difference or I'm missing something.

The difference is that the ignition-cmake available in conda-forge contains a patch to fix the GTS-problem reported by @JShep1 : 
* https://github.com/conda-forge/libignition-cmake0-feedstock/blob/master/recipe/FindGTS.cmake.patch
* https://github.com/conda-forge/libignition-cmake0-feedstock/blob/master/recipe/meta.yaml#L15

So the problem is still present if ignition-cmake is compiled from source when building all the ignition projects together with colcon, but not present if ignition-cmake is installed via conda-forge.

File: tutorials/install.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+1. Clone the repository
+
+    ```
+    git clone https://github.com/ignitionrobotics/ign-common
+    ```
+
+1. Configure and build
+
+    ```
+    cd ign-common; mkdir build;cd build; cmake ..;  make
```
Comment (JShep1) [inline tutorials/install.md:118]:
```suggestion
    cd ign-common
    mkdir build
    cd build
    cmake ..
    make
```

**Review by JShep1 — COMMENTED** [review]
Small consistency nit but LGMT otherwise

**Comment by mabelzhang** [comment]
Same comment about formatting as https://github.com/ignitionrobotics/ign-math/pull/185#issuecomment-756463732
fb6323a tries to fix it.

I'm not sure if that fixes it though. Looking at the profiler tutorial, Troubleshoot section, bullets 2 and 4 have code blocks, one with a preceding empty line and one without, both rendered incorrectly. So the empty line isn't the answer. Do you think changing from 4 spaces to 2 spaces helps??? Notably, the `{.sh}` doesn't help either, because the ign-physics tutorial that renders correctly doesn't have that, these incorrect ones do.
https://ignitionrobotics.org/api/common/3.9/profiler.html

**Comment by mabelzhang** [comment]
Formatting has been fixed, given the tip in the other PR.
So yeah, 2 spaces made a difference!

**Review by JShep1 — APPROVED** [review]

**Review by chapulina — APPROVED** [review]

**Pull request merged**